.github/workflows/release-6.4.yml

```diff
@@ -0,0 +1,106 @@
+on:
+    push:
+      branches: ["v6.4.x"]
+    workflow_dispatch: {}
+  
+permissions:
+    contents: write
+    pull-requests: write
+    id-token: write
+
+name: release-6.4
+  
+jobs:
+    release_please:
+      runs-on: ubuntu-latest
+      outputs:
+        release_created: ${{ steps.release.outputs.release_created }}
+      steps:
+        - id: release
+          uses: googleapis/release-please-action@v4
```

Comment:
This version of release-please is not the same we used on the 6.4 version. We need to add `release-please-config.json` and `.release-please-manifest.json` to the v6.4.x branch

Reply:
Okay, separate PR